nix/shell.nix

```diff
@@ -0,0 +1,63 @@
+{ repoRoot, pkgs, ... }:
+
+cabalProject:
```

Comment:
dead? what is this for?

Reply:
The shell is called for each project variant.
Many repositories pull exes from `cabalProject` into shell.
`plutus` doesn't do it, so I guess we can rename `cabalProject` -> `_cabalProject` to make it more clear.